src/gpodder/gtkui/desktop/podcastdirectory.py

```diff
@@ -450,7 +450,9 @@ def download_data():
             try:
                 podcasts = callback()
             except Exception as e:
-                logger.warning('Got exception while loading podcasts1: %r', e, exc_info=True)
+                logger.warning(
```

Comment:
Is there a reason the warnings should be logged to file and console?

Reply:
no, you're right. I log only when Exception